legacy_packages/sdk/src/evm/core/sdk.ts

```diff
@@ -1921,6 +1922,152 @@ export class ContractDeployer extends RPCConnectionHandler {
     },
   );
 
+  /**
+   * @internal
+   */
+  deployModular = /* @__PURE__ */ buildDeployTransactionFunction(
```

Comment:
since all deployments right now go through `deployContractFromUri` function in legacy packages, can we port all deployment related setup in another PR?

doing it in this PR would be out of context for this PR specific change.